android/src/org/coolreader/crengine/WordTimingAudiobookMatcher.java

```diff
@@ -44,15 +44,25 @@ public WordTimingAudiobookMatcher(File wordTimingsFile, List<SentenceInfo> allSe
 
 	public void parseWordTimingsFile(){
 		List<String> lines = new ArrayList<>();
+		BufferedReader br = null;
 		try {
-			BufferedReader br = new BufferedReader(new FileReader(wordTimingsFile));
+			br = new BufferedReader(new FileReader(wordTimingsFile));
 			String line;
 			while ((line = br.readLine()) != null) {
 				lines.add(line);
 			}
+			br.close();
 		} catch(Exception e) {
 			log.d("ERROR: could not read  word timings file: " + wordTimingsFile + " " + e);
 			lines = new ArrayList<>();
+		} finally {
+			try {
+				if(br != null){
+					br.close();
+				}
+			} catch(Exception e){
+				//ignore
+			}
```

Comment:
why not refactor to try-with-exception paradigm

Reply:
done